azure/Guidelines.md

```diff
@@ -122,14 +140,18 @@ Cloud applications embrace failure. Therefore, to enable customers to write faul
 
 ##### Exactly Once Behavior = Client Retries & Service Idempotency
 
+<a name="http-all-methods-idempotent"></a>
 :white_check_mark: **DO** ensure that _all_ HTTP methods are idempotent.
 
+<a name="http-use-put-or-patch"></a>
 :ballot_box_with_check: **YOU SHOULD** use PUT or PATCH to create a resource as these HTTP methods are easy to implement, allow the customer to name their own resource, and are idempotent.
 
-:heavy_check_mark: **YOU MAY** use POST to create a resource but you must make it idempotent and, of course, the response **MUST** return the URL of the created resource with a 201-Created. One way to make POST idempotent is to use the Repeatability-Request-ID & Repeatability-First-Sent headers (See [Repeatability of requests](#Repeatability-of-requests)).
+<a name="http-post-must-be-idempotent"></a>
+:heavy_check_mark: **YOU MAY** use POST to create a resource but you must make it idempotent and, of course, the response **MUST** return the URL of the created resource with a 201-Created. One way to make POST idempotent is to use the Repeatability-Request-ID & Repeatability-First-Sent headers (See [Repeatability of requests](#repeatability-of-requests)).
 
 ##### HTTP Return Codes
 
+<a name="http-success-status-codes"></a>
```

Comment:
Given section names have changed (you just did it in this PR), maybe we should make explicit bookmarks for headers as well, and then all current children use that as a root to keep them better grouped - at least initially (they may move). E.g.,
```markdown
<a name="http-returns"></a>
##### HTTP Return Codes

<a name="http-returns-sucess"></a>
:white_check_mark: ...
```
Just a thought.

Reply:
I've added anchors for the sections that are the root of the naming convention for rules in that section.